godtools/App/Share/Data/DeepLinkingService/Parsers/KnowGod/KnowGodDeepLinkParser.swift

```diff
@@ -2,28 +2,28 @@
 //  KnowGodDeepLinkParser.swift
 //  godtools
 //
-//  Created by Levi Eggert on 2/22/23.
-//  Copyright © 2023 Cru. All rights reserved.
+//  Created by Levi Eggert on 3/19/25.
+//  Copyright © 2025 Cru. All rights reserved.
 //
 
 import Foundation
 
-class KnowGodDeepLinkParser: DeepLinkUrlParserType {
+class KnowGodDeepLinkParser: DeepLinkUrlParserInterface {
     
     required init() {
         
     }
     
     func parse(url: URL, pathComponents: [String], queryParameters: [String: Any]) -> ParsedDeepLinkType? {
         
-        if pathComponents.first == "lessons" {
-            
+        if pathComponents[safe: 0] == "lessons" {
             return parseLesson(url: url, pathComponents: pathComponents, queryParameters: queryParameters)
         }
-        else {
-         
-            return parseTract(url: url, pathComponents: pathComponents, queryParameters: queryParameters)
+        else if pathComponents[safe: 1] == "tool" {
```

Comment:
Hey @frett made one more change.  KnowGodDeepLinkParser.swift is parsed when the host is ```https://knowgod.com```. Added a check for ```tool``` in the path components to further parse a tool.